src/enums/null-values.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * NullValuesRepr enum defines the format of null, undefined, invalid
+ * values representation in DataModel.
+ *
+ * @readonly
+ * @enum {string}
+ */
+const NullValuesRepr = {
+    NULL: null,
+    NA: 'NA'
+};
```

Comment:
Dont use native format as it might gets mixed with the original data value

either use
```javascript
{
    NULL: '${null}$'
    NA: '${NA}$'
}
```

or 
```javascript
class InvalidAwareType {
    constructor (wrapped, unwrapped) {
         this._wrapped = wrapped;
         this._unwrapped = unwrapped;
    }

    toString () {
        return this._wrapped
    }

     toStringU() {
        return this._unwrapped;
     }
}
```

and then
```javascript
NullAwareType extends InvalidAwareType {
    consturctor (value) {
        super('Null-Value', value);
    }
}
```
so that it can be checked
```
value instanceof NullAwareType
```
Discuss it with @rousan for implementation.

Reply:
> ```js
> {
>     NULL: '${null}$'
>     NA: '${NA}$'
> }
> ```
> Shouldn't the value of `NULL` enum be `null` instead of `"null"` string?